templates/_partials/profile/invitation_link.html.twig

```diff
@@ -0,0 +1,41 @@
+<div class="form rounded-bottom">
+  {% if title is defined %}
+    <h5>{{ title }}</h5>
+  {% endif %}
+  <div class="form-group">
+    <label class="control-label">{{ field_label(form.invitationLink) }}</label>
+    <div class="input-group">
+    <span class="input-group-addon bg-primary"><i class="fa fa-link fa-rotate-90 text-white"></i></span>
+    {{ form_widget(form.invitationLink) }}
+    </div>
+    <div class="d-flex justify-content-end mt-3">
+    <button id="copy-button" type="button" class="btn btn-primary" data-toggle="copy" data-clipboard-text="{{ field_value(form.invitationLink) }}">
+      <i class="fa fa-copy mr-2"></i>
+      {{ 'registration.step.invitation.copy.button'|trans }}
+    </button></div>
+  </div>
+  {% if flow is defined %}
+    {% include '_partials/profile/flow_buttons.html.twig' %}
+  {% endif %}
+</div>
+
+{% block scripts %}
+  <script>
+    document.getElementById('copy-button').addEventListener('click', function(e) {
+      const iconClass = this.childNodes[1].className
+      const text = this.childNodes[2].nodeValue
+
+      this.childNodes[1].className = 'fa fa-check mr-2'
+      this.childNodes[2].nodeValue = "{{ 'registration.step.invitation.link.copied'|trans }}"
+
```

Comment:
I guess you forgot to trigger the copy event, there is a easy API for that.
- https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText
- https://caniuse.com/mdn-api_clipboard_writetext

Reply:
Actually, the copy event is triggered by [ClipboardJS](https://clipboardjs.com/)